Directory.Build.targets

```diff
@@ -4,6 +4,7 @@
   <!-- Central package version management -->
   <PropertyGroup>
     <MoryxFrameworkVersion>6.0.0</MoryxFrameworkVersion>
+    <LangVersion>latest</LangVersion>
```

Comment:
Do we need this in Directory.Build.Props and Directory.Build.targets?

Reply:
Now only in Directoyr.Build.Props